### PR TITLE
blog post for taxonomy community call 

### PR DESCRIPTION
**Pull request by stefaniebutland**

### CODE REVIEW

File: content/blog/2019-03-07-commcall-mar2019.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  - taxize
+  - taxizedb
+  - rgbif
+  - spocc
+  - taxa
+  - metacoder
+  - worrms
+  - wikitaxa
+  - taxview
+  - taxizesoap
```
Comment (sckott) [inline content/blog/2019-03-07-commcall-mar2019.md]: can remove `taxizesoap`. it's more or less abandoned

File: content/blog/2019-03-07-commcall-mar2019.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+Posts about applications of the tools:
+
+- [What are these birds? Complement occurrence data with taxonomy and traits information](https://ropensci.org/blog/2018/09/04/birds-taxo-traits/)
+- [What's this bird? Classify old natural history drawings with R](https://ropensci.org/blog/2018/08/28/birds-ocr/)
+- [Extracting and Enriching Ocean Biogeographic Information System (OBIS) Data with R](https://ropensci.org/blog/2017/01/25/obis/)
+
+Manuscripts:
+
+- Foster ZSL, Chamberlain S and Grünwald NJ. Taxa: An R package implementing data standards and methods for taxonomic data [version 2; referees: 4 approved].  *F1000Research*  2018,  **7** :272 (https://doi.org/10.12688/f1000research.14013.2)
```
Comment (sckott) [inline content/blog/2019-03-07-commcall-mar2019.md]: also this one

> Chamberlain, S. A., & Szöcs, E. (2013). taxize: taxonomic search and retrieval in R. F1000Research, 2, 191. <https://doi.org/10.12688/f1000research.2-191.v2>

Comment (sckott) [reply content/blog/2019-03-07-commcall-mar2019.md]: and 

> Chamberlain, S. A., & Boettiger, C. (2017). R Python, and Ruby clients for GBIF species occurrence data. <https://doi.org/10.7287/peerj.preprints.3304v1>

File: content/blog/2019-03-07-commcall-mar2019.md (hunk 1 above)
Comment (stefaniebutland) [reply content/blog/2019-03-07-commcall-mar2019.md]: done

File: content/blog/2019-03-07-commcall-mar2019.md (hunk 2 above)
Comment (stefaniebutland) [reply content/blog/2019-03-07-commcall-mar2019.md]: done

File: content/commcalls/2019-03-27.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  - spocc
+  - taxa
+  - metacoder
+  - worrms
+  - wikitaxa
+  - taxview
+  - ritis
+  - zbank
+---
+NA
```
Comment (maelle) [inline content/commcalls/2019-03-27.md]:
```suggestion
```

File: content/commcalls/2019-03-27.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+---
```
Comment (maelle) [inline content/commcalls/2019-03-27.md]:
```suggestion
```

you can remove these two last lines, they appeared at some point in the files due to my YAML munging and my not paying enough attention. :-)

**Pull request merged**